deployments/workflows.go

```diff
@@ -116,3 +119,32 @@ func enhanceWorkflows(ctx context.Context, deploymentID string) error {
 	}
 	return nil
 }
+
+// ResolveWorkflowOutputs allows to resolve workflow outputs
+func ResolveWorkflowOutputs(ctx context.Context, deploymentID, workflowName string) (map[string]*TOSCAValue, error) {
+	wf, err := GetWorkflow(ctx, deploymentID, workflowName)
+	if err != nil {
+		return nil, err
+	}
+
+	outputs := make(map[string]*TOSCAValue)
+	for outputName, outputDef := range wf.Outputs {
+		dataType := getTypeFromParamDefinition(ctx, &outputDef)
+
+		res, err := getValueAssignment(ctx, deploymentID, "", "0", "", dataType, outputDef.Value)
```

Comment:
So here we always resolve on instance "0"?

Reply:
Yes, workflow output resolution is not dependent to instances. Am I wrong ?